auth/src/main/java/com/firebase/ui/auth/ui/phone/CountryListLoadTask.java

```diff
@@ -0,0 +1,302 @@
+        countryInfoList.add(new CountryInfo(new Locale("", "SS"), 211));
+        countryInfoList.add(new CountryInfo(new Locale("", "ES"), 34));
+        countryInfoList.add(new CountryInfo(new Locale("", "LK"), 94));
+        countryInfoList.add(new CountryInfo(new Locale("", "SD"), 249));
+        countryInfoList.add(new CountryInfo(new Locale("", "SR"), 597));
+        countryInfoList.add(new CountryInfo(new Locale("", "SJ"), 47));
+        countryInfoList.add(new CountryInfo(new Locale("", "SZ"), 268));
+        countryInfoList.add(new CountryInfo(new Locale("", "SE"), 46));
+        countryInfoList.add(new CountryInfo(new Locale("", "CH"), 41));
+        countryInfoList.add(new CountryInfo(new Locale("", "SY"), 963));
+        countryInfoList.add(new CountryInfo(new Locale("", "TW"), 886));
+        countryInfoList.add(new CountryInfo(new Locale("", "TJ"), 992));
+        countryInfoList.add(new CountryInfo(new Locale("", "TZ"), 255));
+        countryInfoList.add(new CountryInfo(new Locale("", "TH"), 66));
+        countryInfoList.add(new CountryInfo(new Locale("", "TG"), 228));
+        countryInfoList.add(new CountryInfo(new Locale("", "TK"), 690));
+        countryInfoList.add(new CountryInfo(new Locale("", "TO"), 676));
+        countryInfoList.add(new CountryInfo(new Locale("", "TT"), 1));
+        countryInfoList.add(new CountryInfo(new Locale("", "TN"), 216));
+        countryInfoList.add(new CountryInfo(new Locale("", "TR"), 90));
+        countryInfoList.add(new CountryInfo(new Locale("", "TM"), 993));
+        countryInfoList.add(new CountryInfo(new Locale("", "TC"), 1));
+        countryInfoList.add(new CountryInfo(new Locale("", "TV"), 688));
+        countryInfoList.add(new CountryInfo(new Locale("", "VI"), 1));
+        countryInfoList.add(new CountryInfo(new Locale("", "UG"), 256));
+        countryInfoList.add(new CountryInfo(new Locale("", "UA"), 380));
+        countryInfoList.add(new CountryInfo(new Locale("", "AE"), 971));
+        countryInfoList.add(new CountryInfo(new Locale("", "GB"), 44));
+        countryInfoList.add(new CountryInfo(new Locale("", "US"), 1));
+        countryInfoList.add(new CountryInfo(new Locale("", "UY"), 598));
+        countryInfoList.add(new CountryInfo(new Locale("", "UZ"), 998));
+        countryInfoList.add(new CountryInfo(new Locale("", "VU"), 678));
+        countryInfoList.add(new CountryInfo(new Locale("", "VA"), 379));
+        countryInfoList.add(new CountryInfo(new Locale("", "VE"), 58));
+        countryInfoList.add(new CountryInfo(new Locale("", "VN"), 84));
+        countryInfoList.add(new CountryInfo(new Locale("", "WF"), 681));
+        countryInfoList.add(new CountryInfo(new Locale("", "EH"), 212));
+        countryInfoList.add(new CountryInfo(new Locale("", "YE"), 967));
+        countryInfoList.add(new CountryInfo(new Locale("", "ZM"), 260));
+        countryInfoList.add(new CountryInfo(new Locale("", "ZW"), 263));
+        Collections.sort(countryInfoList);
```

Comment:
Copied from digits code, I agree that we should refactor this significantly.  However right now the primary goal of this phone auth implementation is to enable digits customers to migrate so leaning towards "if it ain't broke don't fix it" for the initial release.